p256k1/src/point.rs

```diff
@@ -125,7 +128,7 @@ extern "C" fn ecmult_multi_callback(
             infinity: 0,
         };
 
-        let gej = &(*sp).p[idx].gej as *const secp256k1_gej;
+        let gej = &(*sp).p[idx].gej;
         secp256k1_ge_set_gej(&mut ge, gej);
```

Comment:
Not sure why I chose to use an intermediate here instead of just operating on `*pt` here.  Try changing that, and if the tests pass it'd be nice to save the copy.  But if it doesn't immediately work don't worry about it.